examples/external/Makefile

```diff
@@ -10,7 +10,7 @@
 
 R2IVERSION=0.0
 
-CXXFLAGS:= -std=c++11 $(shell pkg-config --cflags r2inference-$(R2IVERSION)) -pthread
+CXXFLAGS:= -std=c++11 $(shell pkg-config --cflags r2inference-$(R2IVERSION)) -pthread $(shell pkg-config --cflags glib-2.0 gmodule-2.0)
```

Comment:
I think you can do:
CXXFLAGS:= -std=c++11 $(shell pkg-config --cflags glib-2.0 gmodule-2.0 r2inference-$(R2IVERSION)) -pthread